src/server/acl/acl_family.h

```diff
@@ -62,12 +63,154 @@ class AclFamily final {
   // Helper function that loads the acl state of an acl file into the user registry
   GenericError LoadToRegistryFromFile(std::string_view full_path, ConnectionContext* init);
 
+  // Serializes the whole registry into a string
   std::string RegistryToString() const;
 
+  std::string AclCatToString(uint32_t acl_category, User::Sign sign) const;
```

Comment:
are these methods require `this`? I spotted at least some that do not.  Why not put them into the anonymous space of  the cc file?

Reply:
basically anything that does not belong to the class should not be part of its declaration

Reply:
+1

Reply:
This one in particular needs it because it calls  `CategoryToIdx()` which is a method now of AclFamilly. I removed the ones that are independent :)